axoupdater/src/release/github.rs

```diff
@@ -149,27 +149,43 @@ pub(crate) async fn get_github_releases(
     let mut data: Vec<Release> = vec![];
 
     while pages_remain {
+        // fetch the releases
         let resp = get_releases(&client, &url, token).await?;
 
+        // collect the response headers
         let headers = resp.headers();
-        let link_header = &headers[reqwest::header::LINK]
-            .to_str()
-            .expect("header was not ascii")
-            .to_string();
-        pages_remain = link_header.contains("rel=\"next\"");
+        let link_header_val = &headers[reqwest::header::LINK];
```

Comment:
I think the issue here is that the index operation panics if the header isn't there: https://docs.rs/reqwest/0.12.4/reqwest/header/struct.HeaderMap.html#method.index

We should probably use `&headers.get(reqwest::header::LINK)`, which returns an `Option<T>` instead and lets us react properly: https://docs.rs/reqwest/0.12.4/reqwest/header/struct.HeaderMap.html#method.get